specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2020-01-01/SecurityInsights.json

```diff
@@ -0,0 +1,2495 @@
+          "$ref": "#/definitions/TIDataConnectorDataTypes",
+          "description": "The available data types for the connector."
+        }
+      },
+      "type": "object"
+    },
+    "ThreatIntelligence": {
+      "description": "ThreatIntelligence property bag.",
+      "properties": {
+        "confidence": {
+          "description": "Confidence (must be between 0 and 1)",
+          "format": "double",
+          "readOnly": true,
+          "type": "number"
+        },
+        "providerName": {
+          "description": "Name of the provider from whom this Threat Intelligence information was received",
+          "readOnly": true,
+          "type": "string"
+        },
+        "reportLink": {
+          "description": "Report link",
+          "readOnly": true,
+          "type": "string"
+        },
+        "threatDescription": {
+          "description": "Threat description (free text)",
+          "readOnly": true,
+          "type": "string"
+        },
+        "threatName": {
+          "description": "Threat name (e.g. \"Jedobot malware\")",
+          "readOnly": true,
+          "type": "string"
+        },
+        "threatType": {
+          "description": "Threat type (e.g. \"Botnet\")",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "ToggleSettings": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Settings"
+        }
+      ],
+      "description": "Settings with single toggle.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ToggleSettingsProperties",
+          "description": "toggle properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "ToggleSettings"
+    },
+    "ToggleSettingsProperties": {
+      "description": "toggle property bag.",
+      "properties": {
+        "isEnabled": {
+          "description": "Determines whether the setting is enable or disabled.",
+          "type": "boolean"
+        }
+      },
+      "type": "object"
+    },
+    "UebaSettings": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Settings"
+        }
+      ],
+      "description": "Represents settings for User and Entity Behavior Analytics enablement.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/UebaSettingsProperties",
+          "description": "User and Entity Behavior Analytics settings properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "UebaSettings"
+    },
+    "UebaSettingsProperties": {
+      "description": "User and Entity Behavior Analytics settings property bag.",
+      "properties": {
+        "atpLicenseStatus": {
+          "description": "Determines whether the tenant has ATP (Advanced Threat Protection) license.",
+          "enum": [
+            "Enabled",
+            "Disabl
```

Comment:
The enum in ApiVersion only contains 2019-01-01-preview, but in your examples, you are using the value of 2020-01-01. This would cause failure in model validation. Could you help to resolve this problem?

Reply:
Fixed

Reply:
Fixed it